dotcom-rendering/src/layouts/ShowcaseLayout.tsx

```diff
@@ -456,7 +502,9 @@ export const ShowcaseLayout = ({ article, NAV, format }: Props) => {
 									switches={article.config.switches}
 									isAdFreeUser={article.isAdFreeUser}
 									isSensitive={article.config.isSensitive}
-									imagesForAppsLightbox={[]}
+									imagesForAppsLightbox={
```

Comment:
Unrelated to this PR and I know I'm late to the party since I didn't review https://github.com/guardian/dotcom-rendering/pull/8877 but I thought this could be a nice refactoring so writing here:

In [`frontend.ts`](https://github.com/guardian/dotcom-rendering/blob/main/dotcom-rendering/src/types/frontend.ts#L126-L130) we could have:

```
export type DCRArticle = FEArticleType & {
	imagesForLightbox?: ImageForLightbox[] | ImageForAppsLightbox[];
	tableOfContents?: TableOfContentsItem[];
};
```

And in [`article.ts/enhanceArticleType`](https://github.com/guardian/dotcom-rendering/blob/main/dotcom-rendering/src/lib/article.ts#L80-L81)

```
imagesForLightbox: renderingTarget === "Web" ? webLightboxImages(
		data.format,
		enhancedBlocks,
		enhancedMainMedia,
	) : appsLightboxImages(imagesForLightbox)
```

and _I think_ this would be enough for downstream components to know which lightbox images they use. Could be wrong though because this is just an idea and I haven't got my hands dirty in the code! Also there might be other reasons why we chose the two props option that I'm not aware, so just leaving this comment here for the sake of discussion.

Reply:
oh nice! yes that might be a nice improvement! Ideally, `imagesForLightbox` would then be a required prop. I'm not sure what the refactor looks like, but it's worth trying!